src/eureka/S5_lightcurve_fitting/models/BatmanModels.py

```diff
@@ -12,6 +12,73 @@
 from ...lib.split_channels import split
 
 
+class PlanetParams():
+    """
+    Define planet parameters.
+    """
+    def __init__(self, model, pid=0):
+        """ 
+        Set attributes to PlanetParams object.
+
+        Parameters
+        ----------
+        model : object
+            The model.eval object that contains a dictionary of parameter names 
+            and their current values.
+        pid : int; optional
+            Planet ID, default is 0.
+        """
+        # Planet ID
+        self.pid = pid       
+        # Set transit/eclipse parameters
+        self.t0 = None
+        self.rp = None
+        self.inc = None
+        self.a = None
+        self.per = None
+        self.ecc = 0.
+        self.w = None
+        self.fp = None
```

Comment:
Should we have fp default to zero (in the same fashion you adopted for the phase curve Amp... terms)? In the case of the TRAPPIST-1b+c phase curve, we also caught a transit of 1g, so it was mildly annoying to have to add an fp parameter fixed to zero for that planet.

Reply:
Good idea!  Done.